lib/http.js

```diff
@@ -37,43 +36,32 @@ class HttpService {
   start(app) {
     this.app = app;
     if(this.started){
-      for (const index in this.eventConfigurations) {
-        let eventConfiguration = this.eventConfigurations[index];
-        app.registerHTTPDelegate(eventConfiguration.options.path, this);
-      }
+      Object.values(this.eventConfigurations).forEach(
+          eventConfiguration => app.registerHTTPDelegate(eventConfiguration.options.path, this)
+      )
     }
     this.started = true;
   }
+
   handle(req, res, next) {
-    let method = req.method;
-    let path = req.url;
-    let handled = false;
-    for (const index in this.eventConfigurations) {
-      let eventConfiguration = this.eventConfigurations[index];
-      if(eventConfiguration.options.path === path &&  eventConfiguration.options.method === method){
-        console.log("HTTP got called, firing event");
-        handled = this.app.handleEvent(eventConfiguration.id, { "req": req, "res": res });
-      }
-    }
+    const {method, url} = req;
+    const eventConfiguration = Object.values(this.eventConfigurations)
+        .find(({options}) => options.path === url && options.method === method)
 
-    if (!handled){
```

Comment:
We need to return true false, or else the http call hangs if no scripts are connected.

Reply:
okay, I'll revert those